src/microbe_stage/MicrobeHUD.cs

```diff
@@ -243,6 +247,9 @@ public class MicrobeHUD : Node
     /// </summary>
     private float hoverInfoTimeElapsed;
 
+    private System.Collections.Generic.Dictionary<string, ActionButton> abilitiesHotkey =
+        new System.Collections.Generic.Dictionary<string, ActionButton>();
```

Comment:
Why didn't you add a using to the namespace? Why did you use the full name?

Reply:
Jetbrains enforces it, there's also Dictionary usage at line 736 which also specify the full name so I figured this should also follow that.